src/main/java/io/jenkins/plugins/pipelinegraphview/cards/items/SCMRunDetailsItems.java

```diff
@@ -1,50 +0,0 @@
-package io.jenkins.plugins.pipelinegraphview.cards.items;
-
-import io.jenkins.plugins.pipelinegraphview.cards.RunDetailsItem;
-import io.jenkins.plugins.pipelinegraphview.cards.RunDetailsItem.Icon.Ionicon;
-import io.jenkins.plugins.pipelinegraphview.cards.RunDetailsItem.ItemContent;
-import java.util.ArrayList;
-import java.util.List;
-import jenkins.model.Jenkins;
-import jenkins.plugins.git.AbstractGitSCMSource;
-import jenkins.scm.api.SCMRevisionAction;
-import org.jenkinsci.plugins.workflow.job.WorkflowRun;
-
-public class SCMRunDetailsItems {
-
-    public static List<RunDetailsItem> get(WorkflowRun run) {
-        SCMRevisionAction scmRevisionAction = run.getAction(SCMRevisionAction.class);
-        if (scmRevisionAction == null) {
-            return List.of();
-        }
-
-        List<RunDetailsItem> runDetailsItems = new ArrayList<>();
-        boolean githubBranchSourceInstalled = Jenkins.get().getPlugin("github-branch-source") != null;
```

Comment:
Given the GitHub branch source plugin change isn't merged yet I think we should keep this?

You could maybe check its version or even if the new class exists and then not show it if you want to de-risk people getting both?

Reply:
Restored. Haven't done the class check, I'll just update this plugin once GitHub plugin has been updated.